src/style.css

```diff
@@ -0,0 +1,93 @@
+@import url(./styles/star.css);
```

Comment:
Consider using a relative path for importing stylesheets. For example, `@import url('./styles/star.css');`